clients/admin-ui/src/features/user-management/UserForm.tsx

```diff
@@ -101,13 +104,32 @@ const UserForm = ({
     ? ValidationSchema
     : ValidationSchema.omit(["password"]);
 
+  const scopes = useMemo(
+    () => USER_PRIVILEGES.map((policy) => policy.scope),
+    []
+  );
+
+  const selectAllPermissions = useCallback(
+    (formikHelpers: FormikProps<FormValues>) => {
+      formikHelpers.setFieldValue("scopes", [...scopes]);
+    },
+    [scopes]
+  );
+
+  const deselectAllPermissions = useCallback(
+    (formikHelpers: FormikProps<FormValues>) => {
+      formikHelpers.setFieldValue("scopes", [requiredPermission]);
+    },
+    []
+  );
+
   return (
     <Formik
       onSubmit={handleSubmit}
       initialValues={initialValues ?? defaultInitialValues}
       validationSchema={validationSchema}
     >
-      {({ values, setFieldValue }) => (
+      {(formik) => (
```

Comment:
I updated this to the non-destructured variant because it made typing the new helper functions easier.